packages/neon_framework/lib/src/blocs/accounts.dart

```diff
@@ -22,34 +23,40 @@ import 'package:rxdart/rxdart.dart';
 
 const _keyAccounts = 'accounts';
 
-/// Events for the [AccountsBloc].
-@internal
-abstract interface class AccountsBlocEvents {
+/// The Bloc responsible for managing the [Account]s
+// TODO: Make this class properly sealed once we do not need mocks for the Bloc anymore.
```

Comment:
Under what circumstance should we no longer need to mock this bloc?
I'm still not entirely sure that sealed is the best thing.

At least it's not breaking if we decide to change it later on.

Reply:
As far as I understood we want to untangle the Blocs in a way that the blocs don't know about each other. In that case we don't need to mock it anymore. I'm not sure if this will actually work, as all the widgets still use the bloc directly.

Reply:
Let's remove the TODO for now and keep the sealed annotation. When we need to mock other blocs we can do the same thing and just ignore the lint.

Reply:
> I'm not sure if this will actually work, as all the widgets still use the bloc directly.

That's exactly the problem. We might not need it for testing individual blocs anymore but other widgets might still need a mock.

Reply:
Yeah, but I think the sealed annotation is a good compromise for that